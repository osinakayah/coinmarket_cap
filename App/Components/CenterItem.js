import React, { Component } from 'react'
import PropTypes from 'prop-types'
import { View } from 'react-native'
import styles from './Styles/CenterItemStyle'

export default class CenterItem extends Component {
  // Prop type warnings
  static propTypes = {
    children: PropTypes.node.isRequired
  }


  render () {
    return (
      <View style={styles.container}>
        {this.props.children}
      </View>
    )
  }
}
