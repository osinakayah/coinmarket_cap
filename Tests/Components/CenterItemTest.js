
import 'react-native'
import {Text} from 'react-native'
import React from 'react'
import CenterItem from '../../App/Components/CenterItem'
import renderer from 'react-test-renderer'

test('CenterItem component renders correctly', () => {
  const tree = renderer.create(<CenterItem><Text>Hi</Text></CenterItem>).toJSON()
  expect(tree).toMatchSnapshot()
})
