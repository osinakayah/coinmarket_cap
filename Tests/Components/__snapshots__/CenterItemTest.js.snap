// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CenterItem component renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Hi
  </Text>
</View>
`;
